index.html

```diff
@@ -3036,11 +2995,34 @@ <h3><dfn>Navigate To</dfn></h3>
   with <var>session</var> and <a>current top-level browsing
   context</a>.
 
- <li><p>If <var>session</var>&apos;s <a>current top-level browsing context</a> contains
+ <!-- This redirect handling is extremely badly specified -->
+
+ <li><p>While <var>session</var>&apos;s <a>current top-level browsing context</a> contains
   a <a>refresh state pragma directive</a> of <var>time</var> 1 second
-  or less, wait until the refresh timeout has elapsed, a
-  new <a>navigate</a> has begun, and return to the first step of this
-  algorithm.
+  or less, run the following steps:
+
+  <ol>
+    <li><p>Set <var>current URL</var> to <var>session</var>&apos;s
+    <a>current top-level browsing context</a>&apos;s <a>active
+    document</a>&apos;s [=Document/URL=].
+
+   <li><p>Wait until the refresh timeout has elapsed and
```

Comment:
We most likely need a step here to get the `refresh timeout` from the pragma directive.

Reply:
I think making this entire section watertight would be an entire PR on its own.

Reply:
Sure. Given that you didn't change anything here and it is kept as what we had before it's fine to ignore. We all know what to do at this point, and we should spend time on changes that are more urgent for BiDi. So lets close this issue.